### PR TITLE
Add swagger 2.0 API documentation for adapters

### PR DESCRIPTION
**Pull request by t-lichtenstein**

### CODE REVIEW

**Review by fyndalf — APPROVED** [review]

File: adapter-apidoc.yaml [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+basePath: /
+schemes:
+  - http
+  - https
+consumes:
+  - application/json
+produces:
+  - application/json
+paths:
+  /deployChoreography:
```
Comment (friedow) [inline adapter-apidoc.yaml]: Would love to see route names like `POST /choreographies`.

File: adapter-apidoc.yaml [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+        "500":
+          description: Internal Error occured
+          schema:
+            $ref: "#/definitions/ErrorResponse"
+        default:
+          description: Internal Error occured
+          schema:
+            $ref: "#/definitions/ErrorResponse"
+            
+  /enabledTasks:
```
Comment (friedow) [inline adapter-apidoc.yaml]: For this route, I'd like a hirarchy which already specifies the choreography id and a filter to specify the activity state. 
Example: `/choreographies/{choreograpyId}/tasks?state=enabled`

File: adapter-apidoc.yaml [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+        "500":
+          description: Internal Error occured
+          schema:
+            $ref: "#/definitions/ErrorResponse"
+        default:
+          description: Internal Error occured
+          schema:
+            $ref: "#/definitions/ErrorResponse"    
+
+  /executeTask:
```
Comment (friedow) [inline adapter-apidoc.yaml]: Consider the hirarchy here too.
Example: `/choreographies/{choreographyId}/tasks/execute`

**Review by friedow — CHANGES_REQUESTED** [review]

**Review by friedow — APPROVED** [review]
Perfect 🙌

**Pull request merged**